moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -283,7 +283,9 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
   // check that final state is within goal tolerances
   kinematic_constraints::JointConstraint jc(planning_scene->getRobotModel());
   robot_state::RobotState last_state(planning_scene->getRobotModel());
```

Comment:
Initialize from `start_state` directly. Not sure, `req.start_state.joint_state` is complete!
```suggestion
  robot_state::RobotState last_state(start_state);
```

Reply:
@rhaschke You are right. Either way it is better to copy the C++ class instead of the message.

Whether your comment would have been a potential bug in the future depends on the outcome of #1165